packages/workspace/src/generators/workspace-generator/workspace-generator.ts

```diff
@@ -1,42 +1,15 @@
 import { Schema } from './schema';
-import {
-  Tree,
-  formatFiles,
-  generateFiles,
-  names,
-  joinPathFragments,
-  addDependenciesToPackageJson,
-} from '@nrwl/devkit';
-import { nxVersion } from '../../utils/versions';
+import { Tree, stripIndents } from '@nrwl/devkit';
 
 export default async function (host: Tree, schema: Schema) {
-  const options = normalizeOptions(schema);
+  const message = stripIndents`Workspace Generators are no longer supported. Instead,
+    Nx now supports executing generators or executors from local plugins. To get 
+    started, install @nrwl/nx-plugin and run \`nx g plugin\`.
```

Comment:
We need to make `nx g plugin` fit the mold for this.

Currently, it generates e2e tests, and placeholder executors and generators.